power-control-x86/src/power_control.cpp

```diff
@@ -192,11 +186,21 @@ static void setFrontpanelPowerLed(bool led)
         i2c_smbus_write_byte_data(fd, FP_IOX_PORT_REG, portWrite);
         i2c_smbus_write_byte_data(fd, FP_IOX_DATA_REG, dataWrite);
     }
-    else {
+    else
         std::cerr << "Error ioctl call for Front Panel i2c device " << FP_IOX_ADDR << " \n";
-    }
-    if (fd >= 0) {
+
+    if (fd >= 0)
         close(fd);
+
+    return true;
+}
+
+static void setFrontpanelPowerLed(bool led)
+{
+    // check Volcano FP
+    if (i2cCommandLed(VOLCANO_FP_I2C_BUS, led) == false) {
```

Comment:
Lets not create dependency between Volcano and Purico.
if (i2cCommandLed(VOLCANO_FP_I2C_BUS, led) == false)
{
   //print message
}

if (i2cCommandLed(PURICO_FP_I2C_BUS, led) == false)
{
  // Print message
 }

Reply:
done, did not print messages to avoid confusion